input_parser.star

```diff
@@ -134,12 +137,22 @@ DEFAULT_L1_ARGS = {
     # "minimal" preset will spin up a network with minimal preset. This is useful for rapid testing and development.
     # 192 seconds to get to finalized epoch vs 1536 seconds with mainnet defaults
     # Please note that minimal preset requires alternative client images.
-    "l1_preset": "minimal",
+    "l1_preset": "mainnet",
```

Comment:
Interesting. If this passes CI, I think it's okay.

Reply:
I'm afraid we need to keep the `minimal` preset to make the setup of L1 as short as possible.

Reply:
For most use cases, I think `minimal` would make more sense as well. But when testing the L1 deposits/withdrawls specifically with assertoor, this could be changed. But +1 on keeping `minimal` the default value.

Reply:
I understand. We can implement a check to determine if `assertor` is included in the additional services. If it is, we'll override the default `l1_preset` value, setting it to `mainnet`. However, we need to consider an important exception: if the user has explicitly defined an `l1_preset` value in the '`args`, we should respect that choice and use their specified value instead of automatically overriding it.